actionpack/lib/action_dispatch/http/cache.rb

```diff
@@ -80,6 +80,11 @@ def date=(utc_time)
           set_header DATE, utc_time.httpdate
         end
 
+        # By default, Action Dispatch sets all ETags to be weak.
+        # This ensures that if/when the page change semantically,
```

Comment:
This is not a documentation for a method but a comment on how it works in the HTTP point of view. I think this is not the best place to put this kind of information.